Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -636,6 +639,24 @@ namespace AssetProcessor
             return;
         }
 
+        QString absolutePathToFile = jobEntry.GetAbsoluteSourcePath();
+
+        // Write the LFS pointer file related messages to the job log and display it in the job event details UI.
+        AssetProcessor::SetThreadLocalJobId(jobEntry.m_jobRunKey);
```

Comment:
it is not clear why the local thread job ID needs to be set. Does this show up in/with the log file?

Reply:
Without setting the local thread job ID, the log line will only be added to the asset processor log but not the job log, so that people cannot see this error when click on a failed job under the jobs tab. I think adding this error to the job log can make it more obvious to users about the failure since they can see the error in the event log details table directly as shown in the screenshot above.

Reply:
Just updated the code comment to include the reason for setting the thread job ID.